src/Eloquent/Model.php

```diff
@@ -3728,9 +3728,18 @@ public function getQueueableConnection()
      * @param  mixed  $value
      * @return \Illuminate\Database\Eloquent\Model|null
      */
-    public function resolveRouteBinding($value)
+    public function resolveRouteBinding($value, $field=null)
     {
         return $this->where($this->getRouteKeyName(), $value)->first();
     }
 
+    public function getQueueableRelations()
+    {
+        // TODO: Implement getQueueableRelations() method.
+    }
+
+    public function resolveChildRouteBinding($childType, $value, $field)
+    {
+        // TODO: Implement resolveChildRouteBinding() method.
+    }
```

Comment:
not sure whether we should provide implementations for these, or what features they implement.

Reply:
I have not a clue, but they are new model methods that had to be there. We might want to throw BadMethodCall Exceptions to defend against unintended usage until we figure out what they are used for?

Reply:
Oh, also there are merge conflicts in `composer.json`, i suppose we should just override with the version in this PR>